src/localization/dispensingStrings.json

```diff
@@ -96,7 +96,7 @@
     "select_the_prescriber": "Sélectionnez le prescripteur",
     "subtotal": "Sous-total",
     "supplier": "Fournisseur",
-    "total": "Total",
+    "total": "Part du client",
```

Comment:
Just checking this is correct? Translates to "customer share"?

Reply:
I just do whats me told 🤷‍♂ (GTT Told mark to use that who told me to use that!)